serverless-inference/Serverless-Inference-Walkthrough.ipynb

```diff
@@ -13,7 +13,7 @@
     "For this notebook we'll be working with the SageMaker XGBoost Algorithm to train a model and then deploy a serverless endpoint. We will be using the public S3 Abalone regression dataset for this example.\n",
```

Comment:
<p>Can you please delete this sentence? The preview wheels were for re:Invent and are no longer relevant; we're back to pip-installing the latest public versions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3340/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>